versioneer.py

```diff
@@ -2053,7 +2053,7 @@ def do_setup():
             }
         )
 
-    ipy = os.path.join(os.path.dirname(cfg.versionfile_source), "__init__.py")
+    ipy = os.path.join(os.path.dirname(cfg.versionfile_source), "repository.py")
```

Comment:
I suspect that all the changes in this file are unwanted consequences of an automatic Pycharm refactoring and I am afraid that this one in particular could affect some functionalities. Could you please confirm this is actually wanted?

Reply:
Done